controller/control.go

```diff
@@ -135,8 +137,18 @@ func (c *Controller) UpdateVolStatus() {
 		c.ReadOnly = true
 	}
 
-	logrus.Infof("Previously Volume RO: %v, Currently: %v,  Total Replicas: %v,  RW replicas: %v, Total backends: %v",
-		prev, c.ReadOnly, len(c.replicas), rwReplicaCount, len(c.backend.backends))
+	if rwReplicaCount == c.ReplicationFactor {
+		lastSnapshot, err := c.backend.GetLastSnapshot()
+		if err == nil {
+			if err := c.backend.SetCheckpoint(checkpoint); err == nil {
```

Comment:
here, should we be setting `lastSnapshot` as checkpoint right?

Reply:
Fixed it. Typo while splitting PRs.